[
    {
        "content": "<p>Hi <span class=\"user-mention\" data-user-id=\"598112\">@Philip Durbin</span> . I've got another question related to the metadata blocks.</p>\n<p>I've found out that the Citation Metadata Block does not fully comply with the CESSDA DDI 2.5.</p>\n<p>It requires a certain 'date' field that is not being provided.<br>\n<a href=\"/user_uploads/53090/QHDa8ZupjKBy84upwYwotM-W/image.png\">image.png</a><br>\n<a href=\"/user_uploads/53090/oNIDu5gTU0RUyBVcU0dOvK5F/image.png\">image.png</a></p>\n<div class=\"message_inline_image\"><a href=\"/user_uploads/53090/QHDa8ZupjKBy84upwYwotM-W/image.png\" title=\"image.png\"><img src=\"/user_uploads/53090/QHDa8ZupjKBy84upwYwotM-W/image.png\"></a></div><div class=\"message_inline_image\"><a href=\"/user_uploads/53090/oNIDu5gTU0RUyBVcU0dOvK5F/image.png\" title=\"image.png\"><img src=\"/user_uploads/53090/oNIDu5gTU0RUyBVcU0dOvK5F/image.png\"></a></div><p>I've managed to 'trick' the CESSDA validator by feeding it a DDI xml with the &lt;date&gt; instead of &lt;distDate&gt;, but this is a suboptimal solution.</p>\n<p>Is this an issue you were aware of? Should I create an issue :D?</p>",
        "id": 379087285,
        "sender_full_name": "thejmfc",
        "timestamp": 1690452354
    },
    {
        "content": "<p>I asked the developer who fixed up DDI in 5.14 (not out yet) to take a look.</p>",
        "id": 379157855,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690464268
    },
    {
        "content": "<p>Hello, <br>\nWe recently merged the PR <a href=\"https://github.com/IQSS/dataverse/pull/9484\">https://github.com/IQSS/dataverse/pull/9484</a> and it will become part of Dataverse 5.14, which is being released very shortly. The specific goal of that PR was to make Dataverse produce DDI xml that passes the validation by <a href=\"https://cmv.cessda.eu/#!validation\">https://cmv.cessda.eu/#!validation</a>. So I would suggest to upgrade to 5.14 and try again. <br>\nI'm cautiously optimistic that it will resolve your problem. But it is not completely impossible for your datasets to contain some combination of metadata that we somehow missed while testing, that will still result in invalid DDI... so that's the reason I'm not saying that it's \"guaranteed\" to fix it for you. <br>\nIf you run into any problems, please open a new issue and we'll try to fix it asap. We are interested in keeping it valid going forward. <br>\nbest, <br>\n-L.A. <br>\nP.S. It's not shown in your screenshot, but what validation profile are you using? We've been testing under \"CESSDA DATA CATALOGUE (CDC) DDI2.5 PROFILE - MONOLINGUAL: 1.0.4\".</p>",
        "id": 380510086,
        "sender_full_name": "Leo Andreev",
        "timestamp": 1690843315
    },
    {
        "content": "<p><span class=\"user-mention silent\" data-user-id=\"637063\">Leo Andreev</span> <a href=\"#narrow/stream/378866-troubleshooting/topic/CESSDA.20DDI.202.2E5/near/380510086\">disse</a>:</p>\n<blockquote>\n<p>Hello, <br>\nWe recently merged the PR <a href=\"https://github.com/IQSS/dataverse/pull/9484\">https://github.com/IQSS/dataverse/pull/9484</a> and it will become part of Dataverse 5.14, which is being released very shortly. The specific goal of that PR was to make Dataverse produce DDI xml that passes the validation by <a href=\"https://cmv.cessda.eu/#!validation\">https://cmv.cessda.eu/#!validation</a>. So I would suggest to upgrade to 5.14 and try again. <br>\nI'm cautiously optimistic that it will resolve your problem. But it is not completely impossible for your datasets to contain some combination of metadata that we somehow missed while testing, that will still result in invalid DDI... so that's the reason I'm not saying that it's \"guaranteed\" to fix it for you. <br>\nIf you run into any problems, please open a new issue and we'll try to fix it asap. We are interested in keeping it valid going forward. <br>\nbest, <br>\n-L.A. <br>\nP.S. It's not shown in your screenshot, but what validation profile are you using? We've been testing under \"CESSDA DATA CATALOGUE (CDC) DDI2.5 PROFILE - MONOLINGUAL: 1.0.4\".</p>\n</blockquote>\n<p><a href=\"/user_uploads/53090/-dhcZ5UmaDQbz5tNMUDvakU6/2023.08.01.errordate.png\">2023.08.01.errordate.png</a></p>\n<div class=\"message_inline_image\"><a href=\"/user_uploads/53090/-dhcZ5UmaDQbz5tNMUDvakU6/2023.08.01.errordate.png\" title=\"2023.08.01.errordate.png\"><img src=\"/user_uploads/53090/-dhcZ5UmaDQbz5tNMUDvakU6/2023.08.01.errordate.png\"></a></div><p>It might be the Dataverse version, as we're using 5.13. I'll see about an update :) Thanks for the information.</p>",
        "id": 380646075,
        "sender_full_name": "thejmfc",
        "timestamp": 1690886758
    },
    {
        "content": "<blockquote>\n<p>It might be the Dataverse version, as we're using 5.13</p>\n</blockquote>\n<p>It is definitely broken - as in, generates invalid DDI - in 5.13. But 5.14 should now be out in a couple of days.</p>",
        "id": 381310789,
        "sender_full_name": "Leo Andreev",
        "timestamp": 1691018049
    },
    {
        "content": "<p><span class=\"user-mention\" data-user-id=\"622855\">@thejmfc</span> Dataverse 5.14 is out. Please give it a try.</p>",
        "id": 384779218,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1692019833
    },
    {
        "content": "<p><span class=\"user-mention silent\" data-user-id=\"598112\">Philip Durbin</span> <a href=\"#narrow/stream/378866-troubleshooting/topic/CESSDA.20DDI.202.2E5/near/384779218\">disse</a>:</p>\n<blockquote>\n<p><span class=\"user-mention silent\" data-user-id=\"622855\">thejmfc</span> Dataverse 5.14 is out. Please give it a try.</p>\n</blockquote>\n<p>Hi Phil. I think the issue was sorted after we installed 5.14.<br>\nI've changed jobs and are no longer associated with this Dataverse installation.<br>\nI'll still be around though. :)</p>",
        "id": 385649735,
        "sender_full_name": "thejmfc",
        "timestamp": 1692269942
    },
    {
        "content": "<p>Oh! Congrats on the new job! Glad to hear you'll still be around. <span aria-label=\"big smile\" class=\"emoji emoji-1f604\" role=\"img\" title=\"big smile\">:big_smile:</span></p>",
        "id": 385652528,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1692271036
    },
    {
        "content": "<p><span class=\"user-mention silent\" data-user-id=\"598112\">Philip Durbin</span> has marked this topic as resolved.</p>",
        "id": 389625552,
        "sender_full_name": "Notification Bot",
        "timestamp": 1694083440
    }
]